### PR TITLE
Project-setup-practice

### PR DESCRIPTION
**Pull request by mryeminthu**
**Project-setup-practice**
- Use linter
-  Use gitHub flow
- Create a branch
- Add HTML and CSS files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,125 @@
...
+Clone this repository to your desired folder:
+1. Click Code button (shown in green color)
+2. Copy URL.
+3. Please clone it with the following command: git clone git@github.com:myaccount/my-project.git
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (wayungi) [inline README.md]: - [ ] All lines beginning with `>` should be replaced.  The symbol `>` marks a line as a comment and they are only used to provide guidance to you when filling in the `README`. Kindly remove the `>` from your readme 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,125 @@
...
+
+In order to run this project you need: to download it to your desired PC.
+
+### Setup
+
+Clone this repository to your desired folder:
+1. Click Code button (shown in green color)
+2. Copy URL.
+3. Please clone it with the following command: git clone git@github.com:myaccount/my-project.git
+
```
Comment (wayungi) [inline README.md:54]: - [ ] You have excluded many sections under getting started and this has affected the professionalism of your readme. Please do not delete sections of this template. The only optional section is the **FAQs**. You can also from time to time leave out the **tests & deployment** if your project does not have tests & is not deployed 👍🏾 
Screenshot from 2023-05-30 09-16-53

File: README.md [hunk 3]
```diff
@@ -1 +1,125 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (wayungi) [inline README.md]: - [ ] This section serves to give you additional information wen filling in the readme. Kindly always delete it when you are done editing this document.
scrn

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @mryeminthu , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Your PR title is not there yet. We can always make it short by not including **Pull request** in the title because we already know its a PR. I would suggest a title like **Project-setup-practice**. What do you think 🤔 ?
title

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mryeminthu) [reply README.md]: Please let me know how I can change title. (Pull request for set-up-mv branch)

Comment (wayungi) [reply README.md]: Hello @mryeminthu, Changing the PR title is not difficult at all. Go to Pull request on GitHub and look at the title, to the extreme right of this title you should see the edit button. click it and edit your repo. A save button will also appear at about the same position & you should save once you have made the change. Please reach out to me on zoom or slack if you need further clarification 👍🏾

**Review by LionRouge1 — APPROVED** [review]
Hi @mryeminthu, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**